packages/p2p/scripts/__tests__/extract-string.spec.js

```diff
@@ -112,7 +112,7 @@ describe('Integration checks', () => {
             }
         }
 
-        const error_message = `Invalid string format passed to localize/<Localize>:\n\n\t${errors.join('\n\t')}\n\n\t`;
+        // const error_message = `Invalid string format passed to localize/<Localize>:\n\n\t${errors.join('\n\t')}\n\n\t`;
```

Comment:
@iman-fs why do we need to keep it?

Reply:
@maryia-binary  I guess we don't need but since It was not related to my PR changes and still `eslint` was complaining about it, I commented that out. if you think we won't need it later, I can delete the line.

Reply:
@iman-fs we can remove it if it's not used anywhere. is it?🙂

Reply:
sure.